docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+There may be more modifiers in the future. 
+
+Examples: 
+
+1. For the initial of the surname “**_de Souza_**”, in a language that treats the “de” as a tussenvoegsel, the PersonName object can automatically recast `{surname-initial}` to:<br>`{surname-prefix-initial}{surname-core-initial-allCaps} `to get “dS” instead of “d”.
+2. If the locale expects a surname prefix to to be sorted after a surname, then both `{surname-core} `then `{surname-prefix}` would be used as in<br>`{surname-core}, {given} {given2} {surname-prefix}`
+
+## 6 <a name="6-formatting-process" href="#6-formatting-process">Formatting Process</a>
+
+The patterns are in personName elements, which are themselves in a personNames container element. The following describes how these patterns are chosen. If the name locale is different than the formatting locale, then additional processing needs to take place: see [Handling foreign names](#6-8-handling-foreign-names).
+
+The details of the XML structure behind the data referenced here are in [XML Structure](#2-xml-structure).
+
+### 6.1 <a name="6-1-derive-the-name-locale" href="#6-1-derive-the-name-locale">Derive the name locale</a>
+
+If there is no name locale available via the PersonName data interface, then an effective name locale is created based on the characters in the name and the formatting locale. 
+
+* Get the predominant script of the given and surname, and find the likely language as specified by the likely subtags. Use the locale form from the likely language and the script as the name locale.
+
+### 6.2 <a name="6-2-derive-the-formatting-locale" href="#6-2-derive-the-formatting-locale">Derive the formatting locale</a>
+
+If the name locale is different from the formatting locale, and the script of the name is different from the script of the formatting locale, then change the formatting locale to the name locale.
+
+### 6.3 <a name="6-3-derive-the-name-order" href="#6-3-derive-the-name-order">Derive the name order</a>
+
+The name order data provides a mapping from the name locale to an order parameter. In current data, the important part of the name locale is the region code, but allowing a locale code provides for easy extension in the future.`  `
```

Comment:
Probably drop "In current data, the important part of the name locale is the region code, but allowing a locale code provides for easy extension in the future."

Reply:
Done